provider/testdata/Cargo.toml

```diff
@@ -166,12 +166,12 @@ icuexportdata_glob = [
     "collation/implicithan/root_standard_jamo.toml",
     "collation/implicithan/root_standard_meta.toml",
     "collation/implicithan/root_standard_prim.toml",
-    "collation/implicithan/sv_reformed_data.toml",
```

Comment:
If you change these paths, you need to re-run `cargo make testdata-download-sources`